### ISSUE TITLE
Error in v1.3 after autoinstall

### ISSUE THREAD

**Issue by andresgongora** [andresgongora/scripts#77]
This error was reported by drqxr
```
bash: printf: `Y': invalid format character

which: no dig in (/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl)
which: no nslookup in (/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl)
```

- [ ] Fix printf Y
- [x] Fix which
- [x] Add instruction to README for uninstall

Ass soon as all the fixes are implemented on my PC, I will push them to the github repo.

**Comment by andresgongora** [comment]
I think, the first problem is that we are not redirecting the sterr output of getIP*. `which dig > /dev/null` has to become  `which dig > /dev/null 2>&1`

Reff: https://askubuntu.com/questions/474556/hiding-output-of-a-command

**Comment by olmari** [comment]
Hmm... apparently even "which" can behave diffirently on diffirent versions... Indeed in this case finding nothing is an valid result (which needs no spam to tell it). I did left stderr as normal because generally we rather see if there is an actual error, but apparently not here...

For the printf Y I suspect the date format variable and how we carry it around... but only an hunch, not tested yet.

**Comment by andresgongora** [comment]
I was about to write a comment when I saw yours . Indeed, Y suspect the %Y is related to the date. It should be fixed in the new branch I'm working on to aid in the development of #52 . Now date get handled quite differently. I will finish said modifications and push them today if I can; but you can take a look of whats happening in the branch `features/cleanup_status`. But don't get scared, it's not finished yet hehehe

**Comment by andresgongora** [comment]
Pending #80

**Comment by drqxr** [comment]
![Screenshot from 2019-06-06 20-50-16](https://user-images.githubusercontent.com/45234101/59062582-192b7700-889e-11e9-9276-e91dfa7cf4de.png)

Screenshot as promised. Mind you both of the local and external IPv4 are working, this screenshot was done when my PC was offline. Also thank you @andresgongora for sending me the uninstall script, it has worked correctly.

**Comment by andresgongora** [comment]
No worries about the IP. You should never post it anyway ;)

Most of the bugs are fixed (on the yet to be released new version), but we
are still working on that Nast 'Y'.

About the script, you are welcome. It was my fault that I did not add the
instructions to the readme hehehe will also be fixed in the upcoming update
#80

:)

**Comment by drqxr** [comment]
Thanks, much appreciated.

**Comment by olmari** [comment]
@andresgongora I think we could use `prinf --` in the print Y error, according to https://stackoverflow.com/questions/40423203/bash-printf-literal-verbatim-string having -- tells printf that "no more format stuff, all data after this is literal stuff to output", this way printf shouldn't try to intrepet date config params as format variables for itself.

**Comment by olmari** [comment]
I managed to fix the "print Y" problem, pushed fix into features/cleanup_status

**Comment by andresgongora** [comment]
Problem still not solved. Now I get it as well xD
`bash: printf: `Y': invalid format character`

**Comment by olmari** [comment]
For the long route, comment out the 'clear' in status.sh, then run 'source ./secripts/..../status.sh` and you'll get ever line it does annotated on the screen.. from there you can see exactly at where this message occurs for you...

**Comment by andresgongora** [comment]
@olmari My own fault. I was using the old script by mistake. BUG FIXED in #80 as you described.


@drqxr the version 1.4 has just been released. Now the installer is MUCH smarter. Run the following:
```
git clone --recursive https://github.com/andresgongora/scripts.git
chmod +x scripts/install/install.sh
scripts/install/install.sh
rm -fr scripts
```
It will prompt you and isntall/uninstall everything very cleanly
